docs/build/examples/network-inspector/lotus-and-go-ipfs-interactions.md

```diff
@@ -9,7 +9,7 @@ description: This article describes how lotus and go-ipfs nodes interact in gene
 
 [go-ipfs](https://github.com/ipfs/go-ipfs) is a Go implementation of the [IPFS protocol](https://ipfs.io). There are also several IPFS implementations, including [js-ipfs](https://github.com/ipfs/js-ipfs).
 
-lotus and go-ipfs nodes speak several of the same protocols: [libp2p](https://libp2p.io/) for networking, [bitswap](https://docs.ipfs.io/concepts/bitswap/#bitswap) and [graphsync](https://github.com/ipld/specs/blob/master/block-layer/graphsync/graphsync.md) for data transfer, [IPLD](https://ipld.io/) for data structures and formats, and more. Having active lotus and go-ipfs daemons on your machine enables a number of features. For example, you can import and preserve data that is discoverable through the IPFS public network [DHT](https://docs.ipfs.io/concepts/dht/#distributed-hash-tables-dhts) directly into your lotus node and onto the Filecoin network. To learn more about how Filecoin and IPFS interact, read our page on [IPFS and Filecoin](https://docs.filecoin.io/introduction/ipfs-and-filecoin/). To learn about how to enable IPFS/Filecoin node integrations, check out the [lotus docs](https://lotu.sh/en+ipfs-client-integration).
+lotus and go-ipfs nodes speak several of the same protocols: [libp2p](https://libp2p.io/) for networking, [bitswap](https://docs.ipfs.io/concepts/bitswap/#bitswap) and [graphsync](https://github.com/ipld/specs/blob/master/block-layer/graphsync/graphsync.md) for data transfer, [IPLD](https://ipld.io/) for data structures and formats, and more. Having active lotus and go-ipfs daemons on your machine enables a number of features. For example, you can import and preserve data that is discoverable through the IPFS public network [DHT](https://docs.ipfs.io/concepts/dht/#distributed-hash-tables-dhts) directly into your lotus node and onto the Filecoin network. To learn more about how Filecoin and IPFS interact, read our page on [IPFS and Filecoin](https://docs.filecoin.io/introduction/ipfs-and-filecoin/).
```

Comment:
👀  https://docs.filecoin.io/introduction/ipfs-and-filecoin/

Please keep all internal links relative with the `.md` extension